packages/renderer/src/lib/preferences/ProviderActionButtons.spec.ts

```diff
@@ -0,0 +1,493 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import '@testing-library/jest-dom/vitest';
+
+import { render, screen } from '@testing-library/svelte';
+import userEvent from '@testing-library/user-event';
+import { beforeEach, expect, test, vi } from 'vitest';
+
+import type { ProviderInfo } from '/@api/provider-info';
+
+import type { ContextUI } from '../context/context';
+import ProviderUpdateButton from '../dashboard/ProviderUpdateButton.svelte';
+import ProviderActionButtons from './ProviderActionButtons.svelte';
+
+vi.mock(import('../dashboard/ProviderUpdateButton.svelte'));
```

Comment:
```suggestion
import type { ContextUI } from '/@/lib/context/context';
import ProviderUpdateButton from '/@/lib/dashboard/ProviderUpdateButton.svelte';
import type { ProviderInfo } from '/@api/provider-info';

import ProviderActionButtons from './ProviderActionButtons.svelte';

vi.mock(import('/@/lib/dashboard/ProviderUpdateButton.svelte'));

```


suggestion(non-blocking): use path alias

Reply:
updated